reefperf/driver_manager.py

```diff
@@ -0,0 +1,16 @@
+from functools import lru_cache
+from reefperf.cloud_driver import cloud_drivers
+
+
+class DriverManager(object):
+    INSTANCE = None
+
+    @classmethod
+    def get_instance(cls):
```

Comment:
this method should be moved to a `Singleton` class (`reefperf.utils`?), from which `DriverManager` will inherit

Reply:
Also, https://stackoverflow.com/a/33201/95920